packages/components/src/components/post-mainnavigation/post-mainnavigation.scss

```diff
@@ -210,6 +211,10 @@ post-mainnavigation {
           display: none;
         }
       }
+      > a[aria-current='page'] {
+        background-color: #050400;
+        color: #fff;
+      }
```

Comment:
Please add the `[aria-current="page"]` selector to the `mobile-header-interactive` mixin to avoid repeating the styles. 
```suggestion
```